src/control/server/server.go

```diff
@@ -212,14 +213,10 @@ func (srv *server) createEngine(ctx context.Context, idx int, cfg *engine.Config
 
 	// Indicate whether VMD devices have been detected and can be used.
 	cfg.Storage.Bdev.VmdDisabled = srv.bdevProvider.IsVMDDisabled()
+	// Specify path to write output bdev config file.
+	cfg.Storage.Bdev.OutputPath = filepath.Join(cfg.Storage.SCM.MountPoint, bdev.OutConfName)
```

Comment:
Not really crazy about adding new dependencies on the bdev package from the server package. It's also a bit weird to be reaching down into this package for the config file name in order to do the path join out here.

Reply:
Moved this adjustment into config validation and the process of writing the output config is now performed in the bdev provider backend at the end of the storage format process (scm then nvme then write nvme conf) rather than during engine start-up. Writing the nvme conf feels okay to be considered as part of format and the move achieves better separation of bdev details from the server package.